css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 73.5%;
+  top: 5%;
```

Comment:
Instead of pushing an element 73% from the left, why not push it a couple pixels from the right?

Reply:
I thought about that after I sent it and felt very goofy...

Pushed the update now!